### PR TITLE
variation in books: Science Fiction Edition.

### PR DESCRIPTION
**Pull request by faefux**
#### Summary
```SUMMARY: Content "Add sixty science fiction book snippets."```

#### Purpose of change
Continuation of the philosophy book snippets here are some scifi novels.

#### Describe alternatives you've considered
Original list was about 110, so there are other books that can go in if any of these prove contentious or inaccurate.  

I'm still really tempted to add a bunch of these as individual books both so they can be grabbed as quest items and so the titles will also be unique.
#### Additional context
Feel free to add or remove books from this.

### CODE REVIEW

File: data/json/items/book/misc.json [hunk 1]
```diff
@@ -803,7 +803,151 @@
     "intelligence": 6,
     "time": "20 m",
     "chapters": 24,
-    "fun": 3
+    "fun": 3,
+    "snippet_category": [
+      {
+        "id": "scifi1_1",
+        "text": "This is a copy of Gibson's \"Neuromancer\".  Written in the eighties, it was surprisingly accurate in predicting much of modern society..  Until recently."
```
Comment (Night-Pryanik) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This is a copy of Gibson's \"Neuromancer\".  Written in the eighties, it was surprisingly accurate in predicting much of modern society...  Until recently."
```

File: data/json/items/book/misc.json [hunk 2]
```diff
@@ -803,7 +803,151 @@
...
+      },
+      {
+        "id": "scifi1_2",
+        "text": "This is a copy of \"The Stars My Destination\" by Alfred Bester.  \n\nTyger, Tyger, Burning bright,  \nIn the forests of the night:  \nWhat immortal hand or eye,  \nDare frame thy fearful symmetry?"
+      },
+      {
+        "id": "scifi1_3",
+        "text": "This is a copy of  \"The Lathe of Heaven\" by Ursula Le Guin.  Dirty finger-stains have smudged the occasional word."
```
Comment (Night-Pryanik) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This is a copy of \"The Lathe of Heaven\" by Ursula Le Guin.  Dirty finger-stains have smudged the occasional word."
```

File: data/json/items/book/misc.json [hunk 3]
```diff
@@ -803,7 +803,151 @@
...
+      },
+      { "id": "scifi1_4", "text": "This is a copy of  \"The Dispossessed\" by Ursula Le Guin." },
```
Comment (Night-Pryanik) [inline data/json/items/book/misc.json]:
```suggestion
      { "id": "scifi1_4", "text": "This is a copy of \"The Dispossessed\" by Ursula Le Guin." },
```

File: data/json/items/book/misc.json [hunk 4]
```diff
@@ -803,7 +803,151 @@
...
+      },
+      { "id": "scifi1_8", "text": "This is a copy of Philip K. Dick's \"Do Androids Dream of Electric Sheep?\"." },
+      { "id": "scifi1_9", "text": "This is a dog-eared copy of \"Nova Express\" by William Burroughs." },
+      {
+        "id": "scifi1_10",
+        "text": "This is a copy of \"Foundation\" by Isaac Asimov.  The back cover has been ripped off."
+      },
+      {
+        "id": "scifi1_11",
+        "text": "This is a dog-eared copy of \"Dune\" by Frank Herbert.  It has sand between some of it's pages.  Weird."
```
Comment (Night-Pryanik) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This is a dog-eared copy of \"Dune\" by Frank Herbert.  It has sand between some of its pages.  Weird."
```

File: data/json/items/book/misc.json [hunk 5]
```diff
@@ -803,7 +803,151 @@
...
+        "id": "scifi1_30",
+        "text": "This is a copy of \"Grass\" by Sheri S. Tepper.  A child has scribbled over the first pages in crayon."
+      },
+      {
+        "id": "scifi1_31",
+        "text": "This is a copy of William Gibson's \"Count Zero\".  The spine is stamped with 'Library Copy'.  And a sticker reading 'Science Fiction'."
+      },
+      {
+        "id": "scifi1_32",
+        "text": "This is a copy of \"The Thith Season\" by N.K. Jemsin.  It smells faintly of dirt."
```
Comment (Night-Pryanik) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This is a copy of \"The Fifth Season\" by N.K. Jemsin.  It smells faintly of dirt."
```

File: data/json/items/book/misc.json [hunk 6]
```diff
@@ -803,7 +803,151 @@
...
+      },
+      { "id": "scifi1_54", "text": "This is a copy of \"The Day of the Triffids\" by John Wyndham." },
+      { "id": "scifi1_55", "text": "This is a copy of \"A Clockwork Orange\" by Anthony Burges." },
+      { "id": "scifi1_56", "text": "This is a copy of \"The Man Who Fell to Earth\" by Walter Tevis." },
+      { "id": "scifi1_57", "text": "This is a copy of \"Simulacron-3\" by Daniel F. Galouye." },
+      { "id": "scifi1_58", "text": "This is a copy of \"The Glass Bees\" by Ernst Jünger." },
+      { "id": "scifi1_59", "text": "This is a copy of \"Journey to The Center of the Earth\" by Jules Verne." },
+      {
+        "id": "scifi1_60",
+        "text": "This is a copy of Larry Niven's \"Ringworld\".  There are a couple of pages missing from the end of the book..  Luckily only mail-order advertisements."
```
Comment (Night-Pryanik) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This is a copy of Larry Niven's \"Ringworld\".  There are a couple of pages missing from the end of the book.  Luckily only mail-order advertisements."
```

**Comment by faefux** [comment]
<3 @Night-Pryanik

**Comment by jbytheway** [comment]
> I'm still really tempted to add a bunch of these as individual books both so they can be grabbed as quest items and so the titles will also be unique.

In principle we could have more specific mission or dialogue conditions that specified the particular snippet, so that such things were possible even without being made individual items.  Of course, it would be hard for the player to know they've found it.

**Pull request merged**